svgWriterUtils.js

```diff
@@ -41,6 +41,10 @@
         self.write = function (ctx, sOut) {
             ctx.sOut += sOut;
         };
+        
+        self.writeln = function (ctx, sOut) {
+            ctx.sOut += sOut + ctx.terminator;
```

Comment:
I wonder if it would be less change to  error to be simply a call to self.write(ctx, sOut + ctx.terminator) less chance of someone updating write and forgetting writeln?